test/e2e/apimachinery/discovery.go

```diff
@@ -82,7 +82,13 @@ var _ = SIGDescribe("Discovery", func() {
 		}
 	})
 
-	ginkgo.It("should validate PreferredVersion for each APIGroup", func() {
+	/*
+	   Release : v1.19
```

Comment:
I feel it is difficult to merge this into v1.19 because it is necessary to check test stability over 2 weeks as the requirement.

/cc @oomichi